src/main/java/com/sinthoras/hydroenergy/config/HEConfig.java

```diff
@@ -99,6 +101,11 @@ public static void syncronizeConfiguration(java.io.File configurationFile) {
                         "mod. But be warned: you will have limited render capabilities!");
         useLimitedRendering = useLimitedRenderingProperty.getBoolean();
 
+        Property forceOpenGLProperty = configuration.get(Categories.general, "forceOpenGL",
+                Defaults.forceOpenGL, "[CLIENT] Activate this if you REALLY want to force OpenGL. " +
```

Comment:
The variable name and the description tell the same thing in different words. And I have no idea what it actually is. Please specify those.